Configuration/StandardSequences/python/DigiToRaw_Repack_cff.py

```diff
@@ -56,3 +56,23 @@
 DigiToHybridRawRepack = cms.Sequence( DigiToHybridRawRepackTask )
 DigiToVirginRawRepack = cms.Sequence( DigiToVirginRawRepackTask )
 DigiToSplitRawRepack = cms.Sequence( DigiToRawRepackTask, DigiToVirginRawRepackTask )
+
+from EventFilter.SiStripRawToDigi.SiStripDigis_cfi import siStripDigis
+#siStripDigis.ProductLabel = "rawDataRepacker"
+from RecoLocalTracker.Configuration.RecoLocalTracker_cff import siStripZeroSuppression
+
+from RecoLocalTracker.SiStripClusterizer.DefaultClusterizer_cff import *
+siStripClustersHLT = cms.EDProducer("SiStripClusterizer",
+                                    Clusterizer = DefaultClusterizer,
+                                    DigiProducersList = cms.VInputTag(
+                                        cms.InputTag('siStripDigis','ZeroSuppressed'),
+                                        cms.InputTag('siStripZeroSuppression','VirginRaw'),
+                                        cms.InputTag('siStripZeroSuppression','ProcessedRaw'),
+                                        cms.InputTag('siStripZeroSuppression','ScopeMode')),
+                                )
+
+from RecoLocalTracker.SiStripClusterizer.SiStripClusters2ApproxClusters_cfi import *
+SiStripClusters2ApproxClusters.inputClusters = "siStripClustersHLT"
```

Comment:
BTW, import and replace is a bad practice for modules that by construction can be loaded elsewhere. 
clone, preferably with a new name or rely on a process modifier to set the correct value at creation time

Reply:
@mandrenguyen , maybe something like this could be better:

```suggestion
from Configuration.ProcessModifiers.approxSiStripClusters_cff import approxSiStripClusters
approxSiStripClusters.toModify(SiStripClusters2ApproxClusters, inputClusters = "siStripClustersHLT")
```